src/server/controlchan/control_loop.rs

```diff
@@ -1,66 +1,79 @@
-use crate::auth::{Authenticator, UserDetail};
-use crate::metrics;
-use crate::server::chancomms::{InternalMsg, ProxyLoopSender};
-use crate::server::io::*;
-use crate::server::proxy_protocol::*;
-use crate::server::session::SharedSession;
-use crate::server::*;
-use crate::server::{Session, SessionState};
-use crate::storage::{self, ErrorKind};
-use controlchan::codecs::FTPCodec;
-use controlchan::command::Command;
-use controlchan::commands;
-use controlchan::error::{ControlChanError, ControlChanErrorKind};
-use controlchan::handler::{CommandContext, CommandHandler};
-use controlchan::{Reply, ReplyCode};
-
-use futures::channel::mpsc::{channel, Receiver, Sender};
-use futures::{SinkExt, StreamExt};
+use crate::{
```

Comment:
Nice did you do this manually or does vscode have some trick up its sleeve?

Reply:
Visual Studio Code